rails/app/javascript/components/StoryList.jsx

```diff
@@ -80,6 +82,25 @@ class StoryList extends Component {
     const bustCache = () => {
       this.cache.clear(index, 0);
     }
+    const mediaLinkList = 
+      story.media_links.map( function (media_link, linkNum) {
```

Comment:
can you make this a function instead? React paradigm wise, this looks more like a render function.